hasher-matcher-actioner/hmalib/lambdas/api/matches.py

```diff
@@ -272,20 +274,21 @@ def request_signal_opinion_change() -> ChangeSignalOpinionResponse:
             f"Opinion change enqueued for {signal_source}:{signal_id} in {ds_id} change={opinion_change}"
         )
 
-        signal = PDQSignalMetadata(
-            signal_id=signal_id,
-            ds_id=ds_id,
-            updated_at=datetime.datetime.now(),
-            signal_source=signal_source,
-            signal_hash="",  # SignalHash not needed for update
-            tags=[],  # Tags not needed for update
-            pending_opinion_change=pending_opinion_change,
+        signal = ThreatExchangeSignalMetadata.get_from_signal_and_privacy_group(
+            dynamodb_table, signal_id=signal_id, privacy_group_id=ds_id
```

Comment:
nit: cut out `ds_id` here too (okay if you want to leave `bottle.request.query.dataset_q` for now to avoid touching the client)